apps/nextjs-website/src/components/organisms/WebinarsSection/WebinarsSection.tsx

```diff
@@ -50,7 +50,7 @@ const WebinarsSection = ({
         <Box sx={{ display: 'flex', flexDirection: 'column' }}>
           <Box mb={webinars.length ? 6 : 0}>
             <Typography variant='h4' mb={2} color={theme.palette.common.white}>
-              {t(`title.${title}`)}
+              {t(`title.${title}`, { webinars: webinars.length })}
```

Comment:
Maybe you should use a boolean instead of a number and replace the attribute `webinars` with a more meaningful name like `isPlural` (something that makes us understand that the title changes when there's more than a webinar).

Reply:
next-intl accepts only numbers for pluralization: https://next-intl-docs.vercel.app/docs/usage/messages#cardinal-pluralization, one solution could be creating two different lines on translation. What do you think?

Reply:
Proceed with two different lines